modules/va_notify/spec/services/in_progress_form_reminder_spec.rb

```diff
@@ -72,19 +74,25 @@
           described_class.new.perform(in_progress_form.id)
         end
 
-        expect(VANotify::UserAccountJob).to have_received(:perform_async).with('uuid', 'fake_template_id',
-                                                                               {
-                                                                                 'first_name' => 'FIRST_NAME',
-                                                                                 'date' => expiration_date,
-                                                                                 'form_age' => ''
-                                                                               })
+        # rubocop:disable Layout:LineLength
+        expect(VANotify::UserAccountJob).to have_received(:perform_async).with(in_progress_form.user_account_id, 'fake_template_id',
+                                           {
+                                             'first_name' => 'FIRST_NAME',
+                                             'date' => expiration_date,
+                                             'form_age' => ''
+                                           })
+        # rubocop:enable Layout:LineLength
```

Comment:
Is it possible to get this working without disabling the line length? What if you broke the line at `.with(`?

Reply:
Yes it is! Thank you!